lib/dal/src/attribute/prototype.rs

```diff
@@ -301,8 +297,6 @@ impl AttributePrototype {
             for attribute_value_target in workspace_snapshot
                 .incoming_sources_for_edge_weight_kind(target_id, edge_weight_discrim)?
             {
-                // There are also provider edges from the schema variant to the provider.
-                // These should be different edge kinds, I think
```

Comment:
@zacharyhamm I believe this was your comment. Worth deleting?

Reply:
yes, nuke it